src/centengine/engine_impl.cc

```diff
@@ -949,6 +946,40 @@ grpc::Status engine_impl::DeleteDowntimeByHostName(
   return grpc::Status::OK;
 }
 
+grpc::Status engine_impl::DeleteDowntimeByStartTimeComment(
+    grpc::ServerContext* context __attribute__((unused)),
+    const DowntimeStartTimeIdentifier* request,
+    CommandSuccess* response) {
+  time_t start_time;
+  /*hostname must be defined to delete the downtime but not others arguments*/
+  if (!(request->has_start()))
```

Comment:
if it is not optional, why using Uint32Value?
It would be easier to use uint32, wouldn't it?